benchmark.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+FILTER=${1:-Builtin}
+
+mkdir -p ./builtin-benchmark-results
+rm -rf ./builtin-benchmark-results/*
```

Comment:
-r is actually not necessary and it makes it more dangerous

Reply:
fixed

Reply:
fixed